SpatialGDK/Source/SpatialGDKFunctionalTests/SpatialGDK/DynamicSubobjectsTest/DynamicSubobjectsTest.cpp

```diff
@@ -20,29 +24,43 @@
  *true.
  *    - The Server spawns a TestMovementCharacter and makes Client 1 possess it.
  *  - Test:
- *    - Each worker tests if it can initially see the AReplicatedGASTestActor.
+ *    - Each worker tests if it can initially see the ADynamicSubObjectTestActor.
  *    - Repeat the following steps MaxDynamicallyAttachedSubobjectsPerClass + 1 times:
  *		- After ensuring possession happened, the Server moves Client 1's Character to a remote location, so it cannot see the
- *AReplicatedGASTestActor.
- *		- After ensuring movement replicated correctly, Client 1 checks it can no longer see the AReplicatedGASTestActor.
- *		- The Server moves the character of Client 1 back close to its spawn location, so that the AReplicatedGASTestActor is
+ *ADynamicSubObjectTestActor.
+ *		- After ensuring movement replicated correctly, Client 1 checks it can no longer see the ADynamicSubObjectTestActor.
+ *		- The Server moves the character of Client 1 back close to its spawn location, so that the ADynamicSubObjectTestActor is
  *in its interest area.
  *	  - If the "Too many dynamic sub objects" error does not appears in the log the test is successful.
  *  - Cleanup:
  *    - Client 1 repossesses its default pawn.
  *    - The spawned Character is destroyed.
+ *
+ *
+ * A second test case is also tested with this same test.
+ * This tests that
+ * 1. The server adds a dynamic component to the actor
+ * 1. ADynamicSubObjectTestActor moves out of the client's interest
+ * 2. ADynamicSubObjectTestActor has the dynamic component removed
+ * 3. ADynamicSubObjectTestActor moves into the client's interest
+ * 4. The client sees ADynamicSubObjectTestActor no longer has the dynamic component
+ *
+ * This extra test case is implemented in steps 9.1 and 12.1
  */
 
-const static float StepTimeLimit = 10.0f;
+const static float StepTimeLimit = 15.0f;
 
 ADynamicSubobjectsTest::ADynamicSubobjectsTest()
 	: Super()
 {
-	Author = "Evi";
+	Author = "Evi&Arthur&Miron";
 	Description = TEXT("Test Dynamic Subobjects Duplication in Client");
 
 	CharacterSpawnLocation = FVector(0.0f, 120.0f, 40.0f);
 	CharacterRemoteLocation = FVector(20000.0f, 20000.0f, 40.0f); // Outside of the interest range of the client
+
+	TimeLimit = 100.0f;
+	InitialNumComponents = 1;
```

Comment:
maybe this could just be a static constexpr either here or in the header since it's only ever read and never changed?

Reply:
added